<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CBookmarkEditDialog</class>
 <widget class="QDialog" name="CBookmarkEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_general">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_title"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_description">
           <property name="text">
            <string>Description</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_title">
           <property name="text">
            <string>Title</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_favicon">
           <property name="text">
            <string>URL</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_description"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_keywords">
           <property name="text">
            <string>Keywords</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_keywords"/>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLineEdit" name="lineEdit_url"/>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_downloadFavIcon">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_downloadPageInformation">
             <property name="text">
              <string>download information</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="checkBox_favorite">
           <property name="text">
            <string>favorite</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_readItLater">
           <property name="text">
            <string>read it later</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_rating">
           <property name="text">
            <string>Rating</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider_rating">
           <property name="maximum">
            <number>10</number>
           </property>
           <property name="pageStep">
            <number>5</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_rating">
           <property name="maximum">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_trash">
           <property name="text">
            <string>move to trash</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_httpResponseCode">
           <property name="text">
            <string>Code</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_httpResponseCode">
           <property name="minimum">
            <number>-999999</number>
           </property>
           <property name="maximum">
            <number>999999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_httpResponseText">
           <property name="text">
            <string>Reason</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_httpResponseText"/>
         </item>
         <item>
          <widget class="QLabel" name="label_lastCheck">
           <property name="text">
            <string>Last check date</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="dateTimeEdit_lastCheck">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>0</height>
            </size>
           </property>
           <property name="displayFormat">
            <string>yyyy-MM-dd HH:mm:ss</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_checkHttpStatus">
           <property name="text">
            <string>check</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="CCheckedTagView" name="treeView_tags"/>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Screenshot</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QLabel" name="screenshot">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>No screenshot</string>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_comments">
      <attribute name="title">
       <string>Comments</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit_comments"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="metadata_tab">
      <attribute name="title">
       <string>Metadata</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="3" column="0">
          <widget class="QLabel" name="label_metadata_visitCount">
           <property name="text">
            <string>Visit Count</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="spinBox_metadata_visitCount">
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDateTimeEdit" name="dateTimeEdit_metadata_edited">
           <property name="displayFormat">
            <string>yyyy-MM-dd HH:mm:ss</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_metadata_edited">
           <property name="text">
            <string>Edited</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDateTimeEdit" name="dateTimeEdit_metadata_created">
           <property name="displayFormat">
            <string>yyyy-MM-dd HH:mm:ss</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_metadata_created">
           <property name="text">
            <string>Created</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_metadata_lastVisited">
           <property name="text">
            <string>Last Visited</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDateTimeEdit" name="dateTimeEdit_metadata_lastVisited">
           <property name="displayFormat">
            <string>yyyy-MM-dd HH:mm:ss</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_history">
      <attribute name="title">
       <string>History</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QListView" name="listView_history"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QPushButton" name="pushButton_history_remove">
           <property name="text">
            <string>Remove...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_history_removeAll">
           <property name="text">
            <string>Remove all...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CCheckedTagView</class>
   <extends>QTreeView</extends>
   <header>ccheckedtagview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit_title</tabstop>
  <tabstop>lineEdit_description</tabstop>
  <tabstop>lineEdit_keywords</tabstop>
  <tabstop>checkBox_favorite</tabstop>
  <tabstop>checkBox_readItLater</tabstop>
  <tabstop>horizontalSlider_rating</tabstop>
  <tabstop>spinBox_rating</tabstop>
  <tabstop>spinBox_httpResponseCode</tabstop>
  <tabstop>lineEdit_httpResponseText</tabstop>
  <tabstop>dateTimeEdit_lastCheck</tabstop>
  <tabstop>plainTextEdit_comments</tabstop>
  <tabstop>dateTimeEdit_metadata_created</tabstop>
  <tabstop>dateTimeEdit_metadata_edited</tabstop>
  <tabstop>dateTimeEdit_metadata_lastVisited</tabstop>
  <tabstop>spinBox_metadata_visitCount</tabstop>
  <tabstop>listView_history</tabstop>
  <tabstop>pushButton_history_remove</tabstop>
  <tabstop>pushButton_history_removeAll</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CBookmarkEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>567</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CBookmarkEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>567</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_rating</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_rating</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>690</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>800</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_rating</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_rating</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>800</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
